src/neighbor.cpp

```diff
@@ -2774,7 +2768,7 @@ void Neighbor::modify_params(int narg, char **arg)
       }
 
       iarg += 2 + ncollections;
-    } else error->all(FLERR,"Illegal neigh_modify command");
+    } else error->all(FLERR,"Illegal neigh_modify command: {} keyword doesn't exist", arg[iarg]);
```

Comment:
See similar comment about: This would better be "Unknown neigh_modify keyword: {}"